api/app/controllers/spree/api/v2/platform/menus_controller.rb

```diff
@@ -5,6 +5,10 @@ module Platform
         class MenusController < ResourceController
           private
 
+          def resource
```

Comment:
@MatthewKennedy let's add some request specs for this please

Reply:
No problem, just doing the swagger docs so will get the test written shortly.

Reply:
looking through the specs you can filter by location so this is not needed.